Client/Telemetry/TelemetryWrapper.swift

```diff
@@ -155,13 +157,8 @@ class TelemetryWrapper {
         // Save the profile so we can record settings from it when the notification below fires.
         self.profile = profile
 
-        // Get the FxA device id and record it in Glean for the deletion-request ping
-        profile.getCachedClients().upon { maybeClient in
-            guard
-                let deviceId = maybeClient.successValue?.first?.fxaDeviceId.flatMap(UUID.init(uuidString:)) else { return }
-            GleanMetrics.Deletion.fxaDeviceId.set(deviceId)
-        }
-
+        sendFxADeletionPing()
```

Comment:
naming nit: does this actually send the ping, or does it just set the device id? Reading the code makes me think it's the later, in which case something like `setFxADeviceID` may be a better name here.

Reply:
(but also, see my other comment about referring to this thing as something other than "fxa device id" 😄)

Reply:
- To your first comment, this only sends ping and doesn't set anything 😊

- I do agree on changing name, how would you feel about `sendHashedFxADeviceId` ? Reason I want to add `FxA` so that I can distinguish it from ping events.

Reply:
Oops just saw `fxa_hashed_device_id` will use similar `sendFxAHashedDeviceId`

Reply:
To fully avoid any confusion as glean automatically sends event I change name to be `setSyncDeviceId`